Wikipedia/Code/UIColor+WMFStyle.h

```diff
@@ -67,4 +67,20 @@
 
 + (instancetype)wmf_settingsBackgroundColor;
 
+///
+/// @name Derivative Colors
+///
+
+/**
+ *  @return A dimmed copy of the receiver.
+ *
+ *  @see -wmf_colorByScalingComponents:
+ */
+- (instancetype)wmf_colorByApplyingDim;
```

Comment:
maybe a bit more explanation here - maybe some domain knowledge of why we would do this - and maybe rename the method to capture that info.